README.md

```diff
@@ -136,9 +137,7 @@ You should exclude this file from deployment so it can be different on staging a
 
 First you need to be able to log into the dashboard:
 
-```
```

Comment:
Please do not change the markdown code block style as it makes PRs difficult to read and it is our house style to use the ```.

Reply:
(Also the four-spaces style isn't extensible to specify syntax highlighting, although for command line examples that might not come up.)

Reply:
Im not entirely sure, why this happened. Probably just a mistake by me.